src/tinyPages/frame.ts

```diff
@@ -16,10 +16,19 @@
  */
 
 // https://transitory.technology/browser-extensions-and-csp-headers/
```

Comment:
```suggestion
// https://transitory.technology/browser-extensions-and-csp-headers/
// Until this is resolved: https://github.com/w3c/webextensions/issues/483
```

It's worth adding a link to this just in case they fix it.

Also some more discussion in https://groups.google.com/a/chromium.org/g/chromium-extensions/c/ccvoUp7bphw